stdlib/runtime/stringUtils.gr

```diff
@@ -162,6 +162,7 @@ export let rec parseInt = (string: String, radix: Number) => {
       },
       _ => {
         let value = if (negative) value else WasmI64.mul(value, -1N)
+        Memory.incRef(WasmI32.fromGrain(reducedInteger))
```

Comment:
🤭

Reply:
What's with this? Just something you need to do with this new pattern?

Reply:
Yeah, since `reducedInteger` is now a regular-schmegular Grain function now, it needs proper calling convention.